package/network/services/umdns/files/umdns.init

```diff
@@ -46,10 +57,18 @@ start_service() {
 		procd_add_jail umdns ubus log
 		[ -d /etc/umdns ] && procd_add_jail_mount "/etc/umdns"
 	}
+
```

Comment:
I have never seen a mdns service announce itself in the field

Reply:
Updated, kindly review.